mapie/regression/time_series_regression.py

```diff
@@ -96,45 +112,27 @@ def _relative_conformity_scores(
         )
         return scores
 
-    def fit(
+    def partial_fit(
```

Comment:
If I well understood your PR, now if method == "enbpi" then you can either call `.update` or  `.partial_fit`. For now maybe we can keep both, but if we want to keep only the `.update` we could add a DepreciationWarning to partial_fit